spec.bs

```diff
@@ -543,6 +590,8 @@ dictionary AuctionAdConfig {
   record<USVString, record<USVString, double>> perBuyerPrioritySignals;
   Promise<record<USVString, USVString>> perBuyerCurrencies;
   sequence<AuctionAdConfig> componentAuctions = [];
+  Promise<undefined> additionalBids;
+  USVString auctionNonce;

```

Comment:
Why does this have to be a USVString?

Reply:
Agreed, this should be a DOMString. Before that CL, if a developer happens to pass a broken nonce with an unmatched unicode surrogate, each will be replaced with "�". After that CL, these strings are allowed to have unmatched surrogates, but as long as these strings aren't being used for a very specific set of things described by https://w3ctag.github.io/design-principles/#idl-string-types that requires a scalar value string, then it should be fine to use DOMString. So let's update the spec here.

Reply:
Done.